types/retry/index.d.ts

```diff
@@ -0,0 +1,97 @@
+// Type definitions for retry 0.10
+// Project: https://github.com/tim-kos/node-retry
+// Definitions by: Stan Goldmann <https://github.com/krenor>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Options } from './options'
+export * from './options'
+
+export interface RetryOperation {
+    /**
+     * Defines the function that is to be retried and executes it for the first time right away.
+     *
+     * @param {function} callback The function that is to be retried
+     * @param {current} [callback.current] Number of attempts callback has been executed so far.
+     * @param {Options.Attempt} [options]
+     *
+     *
+     * @return {void}
+     */
+    attempt(callback: (current?: number) => void, options?: Options.Attempt): void
+
+    /**
+     * Returns false when no error value is given, or the maximum amount of retries has been reached.
+     * Otherwise it returns true, and retries the operation after the timeout for the current attempt number.
+     *
+     * @param {(Error|any)} [err]
+     *
+     * @return {boolean}
+     */
+    retry(err?: Error | any): boolean
```

Comment:
`X | any` is the same as `any` -- please change to `any` or clarify the intent here so we can help you provide a more accurate definition